tests/osaka/eip7692_eof_v1/eip5450_stack/test_code_validation.py

```diff
@@ -328,6 +347,105 @@ def test_eof_validity(
         possible_exceptions.append(EOFException.STACK_HIGHER_THAN_OUTPUTS)
     if container_has_rjump_off_code:
         possible_exceptions.append(EOFException.INVALID_RJUMP_DESTINATION)
+    if container_has_section_0_retf:
+        possible_exceptions.append(EOFException.INVALID_NON_RETURNING_FLAG)
+
+    eof_test(
+        data=bytes(Container(sections=sections)), expect_exception=possible_exceptions or None
+    )
+
+
+@pytest.mark.parametrize(
+    "inputs", itertools.product(*([possible_inputs_outputs] * (num_sections - 1)))
+)
+@pytest.mark.parametrize(
+    "rjump_kind",
+    RjumpKind.__members__.values(),
```

Comment:
I don't think You don't need the `__members__.values()` construct.  Just use `RjumpKind` (it's cleaner).  This goes for the _whole_ file, there are several places it was used.

```suggestion
    RJumpKind,
```

Reply:
Thx :) Time for gh copilot :joy: turns out I can't even read SO answers carefully.